packages/lodestar-cli/src/util/process.ts

```diff
@@ -0,0 +1,22 @@
+const exitSignals = ["SIGTERM", "SIGINT"] as NodeJS.Signals[];
+
+/**
+ * All CLI handlers should register this callback to exit properly and not leave
+ * a process hanging forever. Pass a clean function that will be run until the
+ * user forcibly kills the process by doing CTRL+C again
+ * @param cleanUpFunction
+ */
+export function onProcessSIGINT(cleanUpFunction: () => Promise<void>): void {
```

Comment:
Not super happy about this function's name. Any suggestion that's better self-describing is super appreciated!

Reply:
just onProcessStop ?